### ISSUE TITLE
DateTimePickerDialog:Talkback stays silent on invoking 'Next/Previous month' button in 'Pick a date & time'

### ISSUE THREAD

**Issue by ss-nitd** [microsoft/fluentui-android#132]
Expected Result:
 Talkback should announce about the month which appears on invoking the 'Next/Previous month' button
Ex: By double tap on Next month button, talkback should announce displayed month(Ex: December 2021)

Actual Result:
Talkback remains silent after hitting on 'Next/Previous month' button

User Impact: ​
Talkback users will get confused to navigate further and access other elements.

https://user-images.githubusercontent.com/18634812/146364219-30a76fd6-cbe0-4c0f-9111-4c759610bec9.mp4

**Comment by joyeeta26** [comment]
This doesn't seem to be fluent DateTimePickerDialog. 
Closing this issue!
Please feel free to reopen with a link to the fluent component. Thanks!
cc: @Dhoot @michaelxiao16
